src/main/java/frc/robot/drive/DriveModes.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.drive;
+
+public class DriveModes {
+    public static DrivePower arcadedrive(double joystickY, double steerAngle, 
+    double currentLeftPower, double currentRightPower, boolean isDrivingForward) {
+
+        double adjustedInput = Functions.calcForwardPower(joystickY);
+        double adjustedSteering = Functions.calcSteeringPower(steerAngle);
+
+        if (joystickY < 0 && adjustedInput > 0) {
+            adjustedInput *= -1;
```

Comment:
You just took the absolute value of joystickY in calcForwardPower, and now we're adding the sign back in? Maybe plot out the expression you're going for and see if you can simplify a bit.